src/pages/events/[eventId]/EventUnRegisterModal.jsx

```diff
@@ -0,0 +1,52 @@
+import { useState } from "react";
```

Comment:
This file should be under the components folder. Right now it is being treated like a page.